controllers/articles.controller.js

```diff
@@ -37,4 +39,33 @@ const getArticleComments = (req, res, next) => {
     });
 };
 
-module.exports = { getArticles, getArticlesById, getArticleComments };
+const postArticleComment = (req, res, next) => {
+  const articleId = req.params.article_id;
+  const username = req.body.username;
+  const commentObj = req.body;
+  const commentBody = req.body.body;
+  if (username && commentBody) {
+    checkUserExists(username)
```

Comment:
fine to do this but sql would require that user exists, as author col contains foreign keys that link to user is in the users table.   So could rely on that to throw an error!